packages/terra-worklist-data-grid/src/subcomponents/Row.module.scss

```diff
@@ -3,12 +3,18 @@
 @import '../clinical-lowlight-theme/Row.module';
 
 :local {
+  tr.selectable:hover {
+    background-color: var(--terra-worklist-data-grid-hover-background-color, rgba(0, 146, 224, 0.12));
```

Comment:
If you are duplicating this style, does it need to be a mixin so that we can ensure consistency